src/IdentityManager/WebApi/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace IdentityManager.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
```

Comment:
Remove dummy controller

Reply:
this is not our part, so as we merged to our branch from master, this class came from identity, and when they fix this we can merge again